spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BraveAutoConfiguration.java

```diff
@@ -244,6 +245,19 @@ BaggagePropagationCustomizer remoteFieldsBaggagePropagationCustomizer() {
 			};
 		}
 
+		@Bean
+		@Order(0)
```

Comment:
We already have a `Order(0)` `BaggagePropagationCustomizer`, i'd prefer `Order(1)`.

Reply:
I have changed the order.